media-libs/libfdk_aac/libfdk_aac-0.1.5.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Fraunhofer AAC codec library"
+DESCRIPTION='The Fraunhofer FDK AAC Codec Library for Android ("FDK AAC Codec") \
+	is software that implements the MPEG Advanced Audio Coding ("AAC") \
+	encoding and decoding scheme for digital audio.'
+HOMEPAGE="https://sourceforge.net/projects/opencore-amr"
+COPYRIGHT="1995-2012 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V."
+LICENSE="Fraunhofer FDK AAC Codec"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/opencore-amr/fdk-aac/fdk-aac-$portVersion.tar.gz"
+CHECKSUM_SHA256="2164592a67b467e5b20fdcdaf5bd4c50685199067391c6fcad4fa5521c9b4dd7"
+SOURCE_DIR="fdk-aac-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="1.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libfdk_aac$secondaryArchSuffix = $portVersion
+	lib:libfdk_aac$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libfdk_aac${secondaryArchSuffix}_devel = $portVersion
+	devel:libfdk_aac$secondaryArchSuffix = $libVersion
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	libfdk_aac$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage libfdk_aac$secondaryArchSuffix \
+	$libDir/libfdk-aac.so.$libVersion
+
+BUILD()
+{
+	runConfigure ./configure --enable-shared
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	install -d -m 755 "$docDir"
+	install -t "$docDir" -m 644 ChangeLog
```

Comment:
I think we should include, in the main package,  the file called NOTICE. Not including it could be seen as an attempt to cheat with the license. Including it, right from the beginning, will help us keep this package in our repo in case upstream finds out we have made a package for it.
```diff
-	install -t "$docDir" -m 644 ChangeLog
+	install -t "$docDir" -m 644 ChangeLog NOTICE \
+		documentation/aacDecoder.pdf \
+		documentation/aacEncoder.pdf
```
Regarding the 2 pdf files, I don't know if this is desirable or not. This is just a suggestion.

Reply:
The license file which came with the port is the NOTICE file. Given that `haikuporter` package this file, I don't see the need to install another copy.

Reply:
Good catch!

Hmm, if I had noticed this was the case then I would have suggested an extra command at the end of INSTALL:
```
	rmdir "$docDir" "$documentationDir/packages"
```
Without this line, the base package has an empty directory. Well this is not an issue. It's just that if someone chooses to install only the base package and goes to /system/documentation/packages/ there will be an empty directory (libfdk_aac_x86 on x86_gcc2, libfdk_aac on x86_64).

To check if a package has empty directories, you can use the "package list" or  "package list -p".
For libfdk_aac_x86 this is what we get:
```
package list -p /system/packages/libfdk_aac_x86-0.1.5-1-x86_gcc2.hpkg  | grep documentation
documentation
documentation/packages
documentation/packages/libfdk_aac_x86
```
That said, there is no need to change the recipe just to remove that empty directory. It can be done next time there is a "real" change, or in case we need to trigger a rebuild for some reason.